clash-prelude/src/Clash/Signal/Delayed/Internal.hs

```diff
@@ -1,7 +1,7 @@
 {-|
   Copyright   :  (C) 2019     , Myrtle Software Ltd.
                      2018     , @blaxill
-                     2018-2019, QBayLogic B.V.
+                     2018-2021, QBayLogic B.V.
```

Comment:
So updating the copyright for QBayLogic isn't what you want to do here, what you want to do is update it with a line for yourself  or the company you work for (whichever is more appropriate). If it's personal then either name or handle starting with @ is good (like the line above the QBayLogic copyright)

I noticed on the other files you updated the copyright on the Myrtle line, this is okay if you work for Myrtle but if not then same applies to those files :eyes: